front-end/src/components/Characteristics/Characteristics.jsx

```diff
@@ -0,0 +1,54 @@
+import './Characteristics.css';
+import { useEffect, useState } from 'react';
+import { productive, creative, innovative, pioneering } from '../../assets/profiles';
+
+const etDescItems = [
+    {id: 0, title: "NĂNG ĐỘNG", img: productive, desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."},
+    {id: 1, title: "SÁNG TẠO", img: creative, desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."},
+    {id: 2, title: "ĐỔI MỚI", img: innovative, desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."},
+    {id: 3, title: "TIÊN PHONG", img: pioneering, desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."},
+];
+
+export const Characteristics = () => {
+
+    const [isVisible, setIsVisible] = useState(false);  
+
+    const handleScroll = () => {  
+        const element = document.querySelector('.characteristics-section');  
+        const position = element.getBoundingClientRect();
+
+        if (position.top < window.innerHeight && position.bottom >= 0) {  
+            setIsVisible(true);  
+        }  
+    };  
+
+    useEffect(() => {  
+        window.addEventListener('scroll', handleScroll);  
+        return () => {  
+            window.removeEventListener('scroll', handleScroll);  
+        };  
+    }, []);
+
+    return (
+        <div className={`characteristics-section ${isVisible ? 'visible' : ''}`}>  
```

Comment:
Suggest create a component Available= ({ when, children }). Then when you want to display a component just use 
` <Available when={isVisible }>
                <p>This content is conditionally rendered!</p>
            </Available>
`
It can resuse for many components